ConsentViewController/Classes/SourcePointClient/SourcePointClient.swift

```diff
@@ -253,13 +253,9 @@ class SourcePointClient: SourcePointProtocol {
                 "siteId": String(propertyId),
                 "consentLanguage": consentLanguage.rawValue
         ])! // swiftlint:disable:this force_unwrapping
-        client.get(urlString: url.absoluteString, apiCode: .CCPA_PRIVACY_MANAGER) { result in
-                handler(Result {
-                    try result.decoded() as CCPAPrivacyManagerViewResponse
-                }.mapError({
-                    InvalidResponseCCPAPMViewEndpointError(error: $0)
-                }))
-            }
+        client.get(urlString: url.absoluteString, apiCode: .CCPA_PRIVACY_MANAGER) {
+            Self.parseResponse($0, InvalidResponseCCPAPMViewEndpointError(), handler)
```

Comment:
meanwhile, gotta go educate myself on the difference between `self` and `Self`

Reply:
@Nevazhnovu `Self` is `self` but for classes. In other words, `Self` refers to `SourcePointClient` while `self`, in this context, would refer to an instance.